README.md

```diff
@@ -0,0 +1,7 @@
+# CQL
+
+This module wraps a Go Cassandra client with sane defaults and additional features: logging, migration management, query builder, etc.
+
+It is mostly used in the storage layers of services.
+
+An example of such stores can be found here: https://github.com/upfluence/tracking-server/tree/staging/storage
```

Comment:
This repo is public and the target URL is private 🙃

Reply:
Oops! I'll add a minimal example then (I wanted to avoid that because snippets also need to be maintained but I guess it makes little sense in that case)

Reply:
The lib interface and construction will likely never change so it's fine to add small snippets here